nef/Algebras/Pasteboard.swift

```diff
@@ -0,0 +1,13 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+import AppKit
+import BowEffects
+
+enum PasteboardError: Swift.Error {
+    case writeToClipboard
+}
+
+protocol Pasteboard {
```

Comment:
I thought I mentioned this in a previous PR, but we should refer to `Clipboard` instead of `Pasteboard` to be consistent.

Reply:
Yep, It is a typo